src/App.tsx

```diff
@@ -1,24 +1,233 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import * as todoService from './api/todos';
+
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
+
 import { UserWarning } from './UserWarning';
+import { ErrorMessages } from './types/ErrorMessages';
+import { TodoList } from './components/TodoList/TodoList';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { ErrorMessage } from './components/ErrorMessage/ErrorMessage';
 
-const USER_ID = 0;
+const USER_ID = 11073;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessages | null>(null);
+  const [filterStatus, setFilterStatus] = useState(Status.ALL);
+  const [todoTitle, setTodoTitle] = useState<string>('');
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
looks like redundant state. for add todo enough tempTodo state.
for deleting todos you extend Todo and add isLoading field.